pear.js

```diff
@@ -23,9 +24,19 @@ const BIN = path.join(PEAR_DIR, 'bin')
 const CURRENT_BIN = path.join(LINK, 'by-arch', HOST, 'bin/pear-runtime' + (isWindows ? '.exe' : ''))
 
 if (isInstalled()) {
-  require('child_process').spawn(CURRENT_BIN, process.argv.slice(2), {
-    stdio: 'inherit'
-  }).on('exit', function (code) {
+  const warning = `Warning! To complete the Pear-Runtime installation, add the following to the beginning of your ${isWindows ? 'Path environment variable' : '$PATH'}:
+${BIN}
+Until then, this request will be forwarded to the internal PEAR binary for you.`
+  console.log(warning)
+  const childProcessExit = new Promise((resolve) => {
+    require('child_process').spawn(CURRENT_BIN, process.argv.slice(2), {
```

Comment:
you should save the child process instance also and .kill() that one in the goodbye